doc/source/index.rst

```diff
@@ -0,0 +1,29 @@
++++++++++++++++++++++++
+ rocPRIM Documentation 
++++++++++++++++++++++++
+
+.. toctree::
+   :hidden:
+
+   intro
+   operations
+
+   devicemodule/index
+   blockmodule/index
+   warpmodule/index
+   threadmodule
+
+   iterators
+   intrinsics
+
+   glossary
+
+``rocPRIM`` is a header-only library providing HIP parallel primitives to ease the maintainability of performant and yet portable GPU-accelerated code on AMD ROCm platform.
+
+.. the source of the following image can be found in the doc/svg directory
+
+.. figure:: static/dna_icon.png
```

Comment:
Sorry I wasn't clear: for the images I meant to remove them entirely for now, not just the SVG sources of them.